effectsize_example.Rmd

```diff
@@ -85,7 +100,7 @@ library(ggstance)   # for geom_pointrangeh()
 
 ```{r boilerplate, include = FALSE}
 set.seed(12)
-format_num <- function(...) format(..., digits = 2)
+format_num <- function(x, k = 2) format(round(x, k), nsmall = k)
```

Comment:
About the trailing 0s after the decimal places, there are two standards:

* Physics: The trailing of 0s indicates the precision of measurement. For example, "0.50" has a margin of error ± 0.005. See [this article](https://en.wikipedia.org/wiki/Accuracy_and_precision#Quantification).

* Psychology: APA's style guide (see [a summary](http://my.ilstu.edu/~jhkahn/apastats.html)) indicates rounding to two decimal places. (Except *p* values)

Therefore, I think that the trailing zeros after the decimal should be kept when measurements are precise enough.